### PR TITLE
ADD product_variant_weight

### PR DESCRIPTION
**Pull request by eLBati**
This module allows to specify different weights for different product variants.
For example, if you have product template A with _weight = 5_ and 2 variants (B and C),
with this module you can set _weight = 6_ for variant B, without changing template A and variant C

### CODE REVIEW

File: product_variant_weight/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Variant weight",
+    "summary": "Use different wieghts for different variants",
```
Comment (tafaRU) [inline product_variant_weight/__openerp__.py]: s/wieghts/weights

File: product_variant_weight/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "category": "Sales Management",
+    "website": "https://www.agilebg.com",
+    "author": "Agile Business Group, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "product",
+    ],
+    "data": [
```
Comment (tafaRU) [inline product_variant_weight/__openerp__.py]: Please remove empty key

**Comment by eLBati** [comment]
@tafaRU thanks, done

**Comment by tafaRU** [comment]
Thanks, now LGTM :+1:

**Comment by hurrinico** [comment]
:+1:

File: product_variant_weight/__init__.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from. import product
```
Comment (qtheuret) [inline product_variant_weight/__init__.py:5]: add a whitespace between 'from' and '.'.

**Comment by eLBati** [comment]
The same module has been added by https://github.com/OCA/product-variant/pull/28
I'm closing this

**Comment by tafaRU** [comment]
@eLBati, strange, your PR was older than other one...

**Pull request closed without merging**